### PR TITLE
feat(UI)/SRT-32: General Review Part 2

### PR DESCRIPTION
**Pull request by FrozenIce0617**
Ticket: [[SRT-32](https://scalio.atlassian.net/browse/SRT-32)]

- Refactoring layout, styling, and responsiveness in Landing page

Loom Video: [Landing Page](https://www.loom.com/share/55b2620be8154e06be18281f239d5b06) (password: keep2021)

### CODE REVIEW

File: src/css/legal.scss [hunk 1]
```diff
@@ -1,129 +1,125 @@
...
+    &:first-child {
+      margin: 0 0 2rem;
     }
-
-    h1 {
-        font-size: 48px;
-        line-height: 52px;
-        letter-spacing: -2px;
-        font-weight: 800;
+  }
```
Comment (astanovferuz) [inline src/css/legal.scss:26]: Extra closing curly brace. I think this should be removed as it's a part of the removed h1 a few lines above.

**Review by astanovferuz — CHANGES_REQUESTED** [review]
I could be wrong, but I think the closing curly brace on line 26 in legal.scss should be removed as it's a part of the removed h1 selector above.

File: src/css/legal.scss (hunk 1 above)
Comment (FrozenIce0617) [reply src/css/legal.scss:26]: Thanks for your review, there is nothing changes in there except replace tab size 4 to 2. :)

**Review by FrozenIce0617 — COMMENTED** [review]
There is nothing changes in there except replacing tab size 4 to 2 by using prettier. :)

**Review by astanovferuz — APPROVED** [review]

**Review by micahjlucas — APPROVED** [review]

**Pull request merged**